app/components/shared/degree_qualification_cards_component.rb

```diff
@@ -34,7 +34,8 @@ def show_institution?(degree)
     # any application choice state
     return true if application_choice_state.nil? || degree.international?
 
-    application_choice_state.to_sym.in? ApplicationStateChange::ACCEPTED_STATES
+    show_institution_states = ApplicationStateChange::ACCEPTED_STATES + ApplicationStateChange::DECISION_PENDING_STATUSES
```

Comment:
I wonder if we can get rid of `show_institution?` check altogether?

Reply:
Thats very true, it works a lot better without the check at all